config/lib/mongoose.js

```diff
@@ -9,11 +9,13 @@ var config = require('../config'),
   mongoose = require('mongoose');
 
 // Load the mongoose models
-module.exports.loadModels = function () {
+module.exports.loadModels = function (callback) {
   // Globbing model files
   config.files.server.models.forEach(function (modelPath) {
     require(path.resolve(modelPath));
   });
+
+  if (callback) callback();
```

Comment:
Let's do this in another PR and keep the promises refactoring contained to the seed functionality.